app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -14,8 +14,8 @@
   <dimen name="state_fragment_split_view_margin_end">32dp</dimen>
   <dimen name="profile_chooser_description_margin_start_profile_not_added">32dp</dimen>
 
-  <dimen name="continue_interaction_item_question_view_margin_end">128dp</dimen>
-  <dimen name="continue_interaction_item_question_view_margin_start">128dp</dimen>
+  <dimen name="continue_interaction_item_question_view_padding_end">128dp</dimen>
+  <dimen name="continue_interaction_item_question_view_padding_start">128dp</dimen>
```

Comment:
Why is this getting changed? This PR is not using this change anywhere.

Reply:
Done.